lib/db/schema/patch-018-019.sql

```diff
@@ -0,0 +1,199 @@
+
+CREATE TABLE devices (
+  uid BINARY(16) NOT NULL,
+  id INT UNSIGNED NOT NULL,
+  sessionTokenId BINARY(32),
+  name VARCHAR(255) NOT NULL,
+  type VARCHAR(16) NOT NULL,
+  createdAt BIGINT UNSIGNED NOT NULL,
+  callbackURL VARCHAR(255),
+  INDEX devices_uid (uid),
```

Comment:
You shouldn't need this index, it's a prefix of the primary key so it's already covered by the main table index.